meinberlin/apps/plans/migrations/0059_migrate_topics_to_m2m_topics.py

```diff
@@ -1,19 +1,36 @@
 # Generated by Django 4.2 on 2023-11-29 13:20
+import logging
 
 from django.db import migrations
 from django.conf import settings
 
+logger = logging.getLogger(__name__)
+
 
 def add_topics_to_m2m_table(apps, schema_editor):
     if hasattr(settings, "A4_PROJECT_TOPICS"):
         plans = apps.get_model("meinberlin_plans", "Plan")
         topic = apps.get_model("a4projects", "Topic")
         for plan in plans.objects.all():
-            for topic_code in plan.topics:
-                plan_topic, _ = topic.objects.get_or_create(
-                    code=topic_code,
+            try:
+                for topic_code in plan.topics.split(","):
+                    if not topic_code:
+                        continue
+                    if len(topic_code) > 3:
```

Comment:
the wrong topics are all longer than 3, so this filters them all out

Reply:
did u see my previous comment, we don't need the check in line 17, the except block will catch it.

Reply:
a rebase is needed bcz of the changelog edit

Reply:
yes, i replied to it on the other pr, it's not caught by the exception afaics

Reply:
about the rebase: do we force push / recreate the v2402.1 release  or do a v2402.2 ?

Reply:
You can force push @goapunk

Reply:
@goapunk what happens to projects that don't have a topic bcz we exclude the wrong topics. Can they be published without a topic?

Reply:
I think if they are published they stay published, otherwise you have to add a topic in the dashboard before you can click on publish